pex/pex_bootstrapper.py

```diff
@@ -547,7 +547,10 @@ def ensure_venv(
                         pex,
                         bin_path=pex_info.venv_bin_path,
                         python=os.path.join(
-                            short_venv_dir, "venv", "bin", os.path.basename(pex.interpreter.binary)
+                            short_venv_dir,
```

Comment:
This is the fix in its entirety. Tested by hand as noted here: https://github.com/pantsbuild/pex/issues/2119#issuecomment-1513946180 but not ITable in any way I can see that is reasonable.